switchsdk/src/androidTest/java/net/gini/switchsdk/uitests/TakePictureScreenTest.java

```diff
@@ -0,0 +1,236 @@
+
+        onView(withText(R.string.exit_dialog_text))
+                .inRoot(isDialog())
+                .check(matches(isDisplayed()));
+
+        onView(withId(android.R.id.button1))
+                .inRoot(isDialog())
+                .check(matches(withText(R.string.confirm)))
+                .check(matches(isDisplayed()));
+
+        onView(withId(android.R.id.button2))
+                .inRoot(isDialog())
+                .check(matches(withText(R.string.cancel)))
+                .check(matches(isDisplayed()));
+
+
+    }
+
+    @Test
+    public void menu_helpShouldDisplayOnboarding() {
+
+        omitOnboarding();
+        startIntent();
+
+        openActionBarOverflowOrOptionsMenu(getInstrumentation().getTargetContext());
+
+        ViewInteraction appCompatTextView = onView(
+                allOf(withId(R.id.title), withText(R.string.menu_entry_help), isDisplayed()));
+        appCompatTextView.perform(click());
+
+        SystemClock.sleep(TIME_TO_WAIT_BETWEEN_STEPS);
+
+        onView(withId(R.id.onBoardingViewPager)).check(matches(isDisplayed()));
+
+    }
+
+    @Test
+    public void menu_shouldContainAllItems() {
+
+        omitOnboarding();
+        startIntent();
+
+        openActionBarOverflowOrOptionsMenu(getInstrumentation().getTargetContext());
+
+        ViewInteraction textView = onView(
+                allOf(withId(R.id.title), withText(R.string.menu_entry_cancel),
+                        childAtPosition(childAtPosition(
+                                IsInstanceOf.<View>instanceOf(android.widget.LinearLayout.class),
+                                0), 0),
+                        isDisplayed()));
+        textView.check(matches(isDisplayed()));
+
+        ViewInteraction textView2 = onView(
+                allOf(withId(R.id.title), withText(R.string.menu_entry_help),
+                        childAtPosition(childAtPosition(
+                                IsInstanceOf.<View>instanceOf(android.widget.LinearLayout.class),
+                                0), 0),
+                        isDisplayed()));
+        textView2.check(matches(isDisplayed()));
+
+    }
+
+    @Before
+    public void setUp() throws Exception {
+        mTargetContext = InstrumentationRegistry.getInstrumentation().getTargetContext();
+        final SharedPreferences sharedPreferences = mTargetContext.getSharedPreferences(
+                OnboardingManager.ONBOARDING_SHARE_PREFERENCES, Context.MODE_PRIVATE);
+        sharedPreferences.edit().clear().apply();
+    }
+
+    @After
+    public void tearDown() {
+        SwitchSdk.getSdk().cleanUp();
+    }
+
+    private void omitOnboarding() {
+        final SharedPreferences sharedPreferences = mTargetContext.getSharedPreferences(
+                OnboardingManager.ONBOARDING_SHARE_PREFERENCES, Context.MODE_PRIVATE);
+        sharedPreferences.edit().putBoolean(OnboardingManager.ONBOARDING_KEY_SHOWN, true).apply();
+    }
+
+    private void startIntent() {
```

Comment:
Maybe extract this in ScreenUtils? You will need it in other test classes in the future.